cysync/src/store/provider/syncProvider/executors/balance.ts

```diff
@@ -113,5 +134,32 @@ export const processResponses = async (
         'Invalid ethCoin found in balance sync item' + item.ethCoin
       );
     }
+  } else if (coin instanceof NearCoinData) {
+    const balanceRes = responses[0];
+
+    const balance = new BigNumber(balanceRes.data);
+    if (item.customAccount) {
+      await customAccountDb.updateBalance({
+        walletId: item.walletId,
+        name: item.customAccount,
+        balance: balance.toString()
+      });
+    } else {
+      const customAccounts = await customAccountDb.getAll({
+        walletId: item.walletId
+      });
+      let totalBalance = new BigNumber(0);
+      for (const customAccount of customAccounts) {
+        totalBalance = totalBalance.plus(new BigNumber(customAccount.balance));
+      }
+      await coinDb.updateTotalBalance({
+        xpub: item.xpub,
+        slug: item.coinType,
+        totalBalance: totalBalance.toString(),
+        totalUnconfirmedBalance: '0'
+      });
```

Comment:
This logic should be on every customAccount balance update.

Reply:
yeap, and this is what is causing 0 balance on adding NEAR for the first time